plugin/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 
 dependencies {
     implementation("com.android.tools.build:gradle:3.0.1")
-    implementation("com.google.apis:google-api-services-androidpublisher:v2-rev77-1.23.0") {
+    implementation("com.google.apis:google-api-services-androidpublisher:v3-rev4-1.23.0") {
```

Comment:
There's already `v3-rev5-1.23.0` available

Reply:
@ChristianBecker They're already up to `rev6`! 😆